dashboard/src/components/api/workspace/che-workspace.factory.ts

```diff
@@ -566,7 +566,7 @@ export class CheWorkspace {
    */
   notifyIfEphemeral(workspaceId: string): void {
     let workspace = this.workspacesById.get(workspaceId);
-    let isEphemeral = workspace && workspace.config && workspace.config.attributes && workspace.config.attributes.persistVolumes ? !JSON.parse(workspace.config.attributes.persistVolumes) : false;
+    let isEphemeral = workspace && workspace.config && workspace.config.attributes && workspace.config.attributes['persistVolumes'] ? !JSON.parse(workspace.config.attributes['persistVolumes']) : false;
```

Comment:
can we still keep `workspace.config.attributes.persistVolumespersistVolumes` as far as this attribute was introduced in IWorkspaceConfigAttributes [here](https://github.com/eclipse/che/pull/12826/files#diff-eef3e1d4b54658f9d78c999b924ec5f9R323)

Reply:
Missed that one, fixed now.